ykcompile/src/lib.rs

```diff
@@ -789,7 +666,6 @@ impl<TT> TraceCompiler<TT> {
         let save_regs = [RDI, RSI, RDX, RCX, R8, R9, R10, R11]
```

Comment:
The comment above this is inaccurate. Shall we fix it now?

Reply:
Yes, I think we should fix it.